unit-tests/functional/utilities.h

```diff
@@ -0,0 +1,97 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2020 Intel Corporation. All Rights Reserved.
+
+#include "../tests.h"
+#include "../include/librealsense2/rs.hpp"
```

Comment:
I don't quite understand why `../include/` is needed... isn't the include dir already in the include list?

Reply:
done